src/components/dropdown/dropdown.tsx

```diff
@@ -18,12 +18,12 @@ export interface DropDownState {
 
 require('./dropdown.scss');
 
-const dropDownOpened = new BehaviorSubject<DropDown>(null);
+const dropDownOpened = new ReplaySubject<DropDown>(1);
```

Comment:
Both this and the old code have a small memory issue which I'll file an issue for separately.  Basically, even after the dropdown has been removed by React, this subject will end up retaining it in memory until a new dropdown is opened.